src/ci/docker/dist-x86-linux/Dockerfile

```diff
@@ -86,7 +46,7 @@ ENV RUST_CONFIGURE_ARGS \
       --host=$HOSTS \
       --enable-extended \
       --enable-sanitizers
-ENV SCRIPT python2.7 ../x.py dist --host $HOSTS --target $HOSTS
+ENV SCRIPT scl enable python27 devtoolset-6 python2.7 ../x.py dist --host $HOSTS --target $HOSTS
```

Comment:
I'm not completely sure, but as I understand this `scl enable` sets up the environment to use the specified tools? In that case this may not work correctly since the `./configure` script is ran outside of that environment, I think.